src/main/java/org/rsa/net/apis/wandbox/models/CompileResultModel.java

```diff
@@ -0,0 +1,19 @@
+package org.rsa.net.apis.wandbox.models;
+
+public record CompileResultModel(
+        String status,
+        String compiler_output,
+        String compiler_error,
+        String compiler_message,
+        String program_output,
+        String program_error,
+        String program_message
+) {
+    public String getStatus() {
+        return switch (status) {
+            case "0" -> "Success";
+            case "1" -> "Failed";
+            default -> "Unknown";
```

Comment:
These could be enums

Reply:
The concern around using enums is if they change the API at all, we can't guarantee we won't be giving invalid responses. This way we know for a fact we are supporting 0 / 1 and anything else has a fail-safe of "Unknown".

Reply:
Aren't we the ones creating the CompileResultModel though. It can go through a transformer from raw response to our more defined object on being received. I'm not familiar with the entire flow though yet.